sso-kit-starter-hilla/src/main/java/com/vaadin/hilla/sso/starter/SingleSignOnConfiguration.java

```diff
@@ -162,6 +163,8 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
             // Disable CSRF for Back-Channel logout requests
             final var matcher = backChannelLogoutFilter.getRequestMatcher();
             http.csrf().ignoringRequestMatchers(matcher);
+        } else {
```

Comment:
Until now, the back-channel logout was disabled by default, it could be enabled by a property. Does this mean that from now it is enabled always?

Reply:
Yes. If the deprecated filter is not enabled explicitly, the Spring one will handle Back-Channel Logout requests.

Reply:
Created a ticket for tracking the updates https://github.com/vaadin/sso-kit/issues/163